### DISCUSSION TITLE
Listing caption is missing a space

### DISCUSSION THREAD

**Discussion by PierreMarchand20** [pradyunsg/furo#522] [Potential Issue]
When using `numfig = True` in `config.py` and a code-block like

```
.. code-block:: cpp
    :caption: main.cpp
    :name: main_multiple_file

    #include <iostream>
    int main(){
        std::cout << "Hello world!\n";
        return 0;
    }
```

there is no space between `Listing x.y.z` and the caption:
![image](https://user-images.githubusercontent.com/15056323/188835582-3c2d4adb-eefe-44a3-9066-e915603dbdf7.png)

**Comment by umarcor** [comment]
I came across this same issue. This is a reproducer using a fork of this repo:

- https://github.com/umarcor/furo/commits/umarcor/numfig
  - https://github.com/umarcor/furo/commit/eda7aff84bb44015520472fa59269e55fc7255bd
- Figures and Tables look good:
  - https://umarcor.github.io/furo/umarcor/numfig/kitchen-sink/images/#figures
  - https://umarcor.github.io/furo/umarcor/numfig/kitchen-sink/tables/#list-tables
- But code blocks are missing a space:
  - https://umarcor.github.io/furo/umarcor/numfig/kitchen-sink/blocks/#id1
  - https://umarcor.github.io/furo/umarcor/numfig/kitchen-sink/blocks/#with-caption

**Comment by simaoafonso-pwt** [comment]
Got hit with this. The workaround is relatively simple.

1. On `conf.py`, add:

```python
html_css_files = ['custom.furo.css']
```

2. On `_static/custom.furo.css`, write:

```
.caption-number {
  margin-right: .5em;
}
```

This should really be fixed, it's a small change.
